include/groonga/expr.h

```diff
@@ -96,6 +96,38 @@ grn_expr_append_const_float(
 GRN_API grn_rc
 grn_expr_append_op(grn_ctx *ctx, grn_obj *expr, grn_operator op, int nargs);
 
+/**
+ * \brief Extract keywords from an expression.
+ *
+ * For example usage:
+ * ```c
+ *  grn_obj keywords;
+ *  GRN_PTR_INIT(&keywords, GRN_OBJ_VECTOR, GRN_ID_NIL);
+ *  // You don't need to unlink keyword because it is owned by \p expr.
+ *  grn_expr_get_keywords(ctx, expr, &keywords);
+ *  {
+ *    int i, n_keywords;
```

Comment:
`size_t` is better because `GRN_BULK_VSIZE(&keywords) / sizeof(grn_obj *)` will be `size_t`:

```suggestion
 *    size_t i, n_keywords;
```